project/Build.scala

```diff
@@ -170,8 +170,8 @@ object Build extends sbt.Build {
       val ScalaTest = "3.0.5"
       val ScalaCheck = "1.13.4"
       val MacWire = "2.2.5"
-      val Marathon = "1.6.322"
-      val MarathonPluginInterface = "1.6.322"
+      val Marathon = "1.6.340"
```

Comment:
let's not update marathon in this PR and do it in a separate one. We need the 1.6.322 artifacts anyway